### ISSUE TITLE
error on missing packages check, libxml-2.0, openssl and json-c, even though packages are installed

### ISSUE THREAD

**Issue by Corinna1** [alip/pinktrace#27]
I receive the following error (same error as jwilk 4 years ago) even after installing check (version 0.9.9). 

_checking for CHECK... no
configure: error: Package requirements (check >= 0.9.4) were not met:

No package 'check' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables CHECK_CFLAGS
and CHECK_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details._

I tried commenting out the line PKG_CHECK_MODULES(CHECK, check >= 0.9.4) in configure.ac, but then got a similar error for three other packages libxml-2.0, openssl and json. Again these packages (or newer versions of them) are installed. 

I am working with a CentOS 7 installation. Please advise on these errors.

**Comment by Corinna1** [comment]
After intense research I found the packages required to get it compiled. However, the initial error on the package 'check', I still worked around by commenting out PKG_CHECK_MODULES(CHECK, check >= 0.9.4) in configure.ac

Now here is how I resolved the other errors  I encountered (on CentOS 7): First of all the packages autoconf, automake and libtool are needed to get started. Then I installed the following packages:

libxml2-devel   (to resolve "No package 'libxml-2.0' found")
openssl-devel  (to resolve "No package 'openssl' found")
json-c-devel (to resolve "No package 'json' found")
libattr-devel (to resolve "configure: error: libattr1 missing")

After installing these development packages the autogen.sh script ran through.
